src/views/Events/EventsFilter/index.tsx

```diff
@@ -169,7 +169,7 @@ function EventsFilter(props: EventsFilterProps) {
     } = useQuery<EventOptionsForFiltersQuery>(EVENT_OPTIONS);
 
     const violenceOptions = data?.violenceList?.results;
-    const disasterCategoryOptions = data?.disasterCategoryList?.results;
+    const disasterCategoryOptions = data?.disasterSubTypeList?.results;
```

Comment:
```suggestion
    const disasterSubTypeOptions = data?.disasterSubTypeList?.results;
```

We will need to modify the form and query itself!